src/model/channel/message.rs

```diff
@@ -72,6 +72,13 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    /// **Note**:
```

Comment:
Just a small blank line here so the note is on it's own line.  (Markdown needs 2 blank lines to actually make a line break)

```suggestion
    ///
    /// **Note**:
```